samwise.js

```diff
@@ -12,15 +12,14 @@
     var autoplayDefault = 0;
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.